lib/messagebird/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module MessageBird
   module Version
-    STRING = '2.0.1'
```

Comment:
This MR is only introducing new features to the gem, which is a minor upgrade. Why we're doing a major upgrade here?

Reply:
See https://github.com/messagebird/ruby-rest-api/pull/62#discussion_r660814601